documents/motions/10.2024.9 - Regulations Committee.md

```diff
@@ -19,3 +19,6 @@ The WCA Regulations Committee is an Advisory Committee of the WCA with the follo
 6. Decision making on the interpretation of the Regulations, if any unforeseen, uncovered, or unresolved incident happens during WCA Competitions.
 7. Advising the WCA Community on the correct application of the Regulations.
 8. Encouraging members of the WCA Community to create and maintain translations of the Regulations, and steering the process of publishing those translations.
+9. Reducing conflicts of roles:
```

Comment:
motion needs slight restructuring - this provision should not be in the "rights and Duties" of WRC, it should be its own sub-point